examples/tutorials/advanced/Federated Recurrent Neural Network.ipynb

```diff
@@ -18,7 +18,7 @@
     "\n",
```

Comment:
<p>you could change this to: <code>!pip install -r "../../../requirements.txt"</code> which runs on the notebook ;)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/2613/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
Done in the latest commit!